### PR TITLE
fix(Export): RHICOMPL-1723 - Readd export to reducer

### PR DESCRIPTION
**Pull request by bastilian**
This go accidentally removed with https://github.com/RedHatInsights/compliance-frontend/pull/1152

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: src/store/Reducers/SystemStore.js [hunk 1]
```diff
@@ -216,5 +217,9 @@ export const systemsReducer = (INVENTORY_ACTION, columns) => applyReducerHash({
     },
     ['SELECT_ENTITIES']: (state, { payload: { ids } }) => ({
         selectedEntities: ids
-    })
+    }),
+    [EXPORT]: (state, { payload: { format } }) => {
+        exportFromState(state, format);
```
Comment (vkrizan) [inline src/store/Reducers/SystemStore.js:222]: This does not yield results. I guess the use of our own get systems from GraphQL is not compatible with a general export. :confused:

**Review by vkrizan — COMMENTED** [review]
Thank you!

I guess this was missing since we switched to new InventoryTable component. Partially we captured it after UX review at RHICOMPL-1662.

File: src/store/Reducers/SystemStore.js (hunk 1 above)
Comment (bastilian) [reply src/store/Reducers/SystemStore.js:222]: @vkrizan Odd, it works fine here:

https://user-images.githubusercontent.com/7757/115771483-45771080-a3ae-11eb-9c47-d8bd01fb5e99.mp4

Is it on any specific instance of the `InventoryTable`?

Comment (vkrizan) [reply src/store/Reducers/SystemStore.js:222]: Oh, I see that exports only after a system is selected. From time to time upon load it exports nothing until there is a state change in selection. It seems to happen when the double refresh is visible.

Comment (bastilian) [reply src/store/Reducers/SystemStore.js:222]: @vkrizan We can wait till other changes are merged to master that address the "double refresh" if you want to confirm it is working then too.

**Review by vkrizan — APPROVED** [review]
Thanks.

**Comment by bastilian** [comment]
https://github.com/RedHatInsights/compliance-frontend/pull/1206 makes this obsolete.

**Pull request closed without merging**